### PR TITLE
Fixed some warnings and errors in docs.

### PR DESCRIPTION
**Pull request by bigcat88**
Fixes 3 Warnings and 3 Errors(all in **filesystem.rst**)

### CODE REVIEW

File: developer_manual/digging_deeper/continuous_integration.rst [hunk 1]
```diff
@@ -105,7 +105,7 @@ Afterwards you'll be able to check the app with the following ``psalm.xml`` that
 
 
 You can put this process into a GitHub Action that is run for every pull request.
-Check our `simple github action <https://github.com/nextcloud/.github/blob/master/workflow-templates/psalm.yml>`_ from
+Check our `psalm github action <https://github.com/nextcloud/.github/blob/master/workflow-templates/psalm.yml>`_ from
```
Comment (bigcat88) [inline developer_manual/digging_deeper/continuous_integration.rst:108]: This removes Warning about that we use `simple github action` twice with different urls

File: developer_manual/basics/storage/filesystem.rst [hunk 2]
```diff
@@ -24,7 +24,6 @@ Access to the filesystem is provided by the ``IRootFolder`` which can be injecte
 From the root folder you can either access a user's home folder or access a file or folder by its absolute path.
 
 .. code-block:: php
-    <?php
```
Comment (bigcat88) [inline developer_manual/basics/storage/filesystem.rst:27]: these block were absent in documentation due to parsing error, with these changes it is ok

**Review by andrey18106 — APPROVED** [review]
👌

**Pull request merged**